app/controllers/proposals_controller.rb

```diff
@@ -27,11 +27,11 @@ def index
     @pending_data = listing.pending
     @pending_review_data = listing.pending_review
     @completed_data = listing.completed.alter_query { |rel| rel.limit(@closed_proposal_limit) }
-    @canceled_data = listing.canceled
+    @canceled_data = listing.canceled.alter_query { |rel| rel.limit(@closed_proposal_limit) }
```

Comment:
thoughts on putting `alter_query { |rel| rel.limit(@closed_proposal_limit) }` in a method that we can call here? That way we can have a test for this logic without needing a controller spec.